src/sagas/profile.js

```diff
@@ -4,6 +4,10 @@ import { getPangeaLibrary } from '../services/container';
 import { REQUEST_PROFILE_UPDATE, DONE_USER_EDITING, SET_USER_PROFILE, REQUEST_GET_PROFILE, CANCEL_USER_EDITING } from '../actions/profile';
 
 export const getProfileState = state => state.profile;
+/**
+ * @desc function generator for profile update saga
+  * @return {void}
```

Comment:
Did you use different space count here intentionally? Just wanted to check :slightly_smiling_face: